tests/non_search/setup_helpers.py

```diff
@@ -147,6 +149,35 @@ def setup_with_two_docs(db: Session):
     )
 
 
+def setup_with_documents_large_with_families(
+    documents_large: list[Dict],
+    db: Session,
+):
+    # Collection
+    collection1, collection2 = get_default_collections()
+    add_collections(db, collections=[collection1])
+
+    # Family + Document + events
+    family1, family2 = get_default_families()
+
+    family1["metadata"] = {
+        "topic": "Mitigation",
+        "sector": "Economy-wide",
+    }
+
+    family1["documents"] = documents_large[:20]
+    family2["documents"] = documents_large[20:]
```

Comment:
suggestion: This split feels a bit magic to me, maybe adding a variable for it would be nice?

Reply:
✅ That was a bit lazy of me.